Backend/ServiceLayer/Services/EmailService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MimeKit;
+using MailKit;
+using MailKit.Net.Smtp;
+
+
+namespace ServiceLayer.Services
+{
+    public class EmailService: IEmailService
+    {
+        //Need to setup email server before populating these variables with data
+        private const string SmtpServer = "email-smtp.us-west-2.amazonaws.com";
+        private const int SmtpPort = 587;
+        private const string SmtpUsername = "AKIAIKGDZB4JHYJW4NHQ";
+        private const string SmtpPassword = "BDREGCLUMHfpqXy36czX4B9zQre6IbvCx4CausIn3pgQ";
```

Comment:
These variables should come from the environment. By committing these, you've just made these public to the web. Anyone can now send mail from your AWS account. Please remove these from source and then *reset* these credentials in your AWS console.

In future, all configs should *not* be hardcoded. They *must* come from the environment.

This is **urgent**